src/graph.ts

```diff
@@ -511,38 +515,43 @@ export class TransferNode {
     }
 
     /**
-     * Return the transfer rate into the output container
-     * This must be the minimum of
-     * (total consumption rate leaving output, maximum transfer unit rate)
+     * Return the transfer rate into the output container.
      */
     get outflowRate(): PerMinute {
         let consumptionRate = 0
         if (this.output !== undefined) {
             for (const consumer of this.output.consumers) {
-                if (consumer instanceof IndustryNode) {
+                if (isIndustryNode(consumer)) {
+                    // add the industry consumer input rate
                     consumptionRate += consumer.getInput(this.item)
                 } else {
-                    consumptionRate += consumer.inflowRatePerContainer
+                    // the only time we get here is when the output's consumer is a transfer unit,
+                    // in which case it is simply moving byproduct.
+                    consumptionRate += 0
```

Comment:
Is this intentional? That's a no-op.

Reply:
Yes, just to make it clear that the movement of byproduct is not counted in the egress of a container. We may wish to change this in the future.